admin/filters/class-cornerstone-filter.php

```diff
@@ -32,7 +44,7 @@ public function filter_posts( $where ) {
 
 			$where .= sprintf(
 				' AND ' . $wpdb->posts . '.ID IN( SELECT post_id FROM ' . $wpdb->postmeta . ' WHERE meta_key = "%s" AND meta_value = "1" ) ',
-				WPSEO_Meta::$meta_prefix . WPSEO_Cornerstone::META_NAME
+				WPSEO_Meta::$meta_prefix . 'is_cornerstone'
```

Comment:
Is there a specific reason why we don't have a class constant or method to provide this value?
Having a string for this increases the risk of typos and inconsistent implementations.